js/classes/entities/background/decoration.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * General-purpose class for displaying entites that do not move and which cannot be collided with.
+ * 
+ * @author Trae Claar
+ */
+class Decoration {
+
+    /**
+     * Constructor for a Decoration. The Decoration is positioned with respect to the BOTTOM (not top) left 
+     * corner.
+     * 
+     * @param {string} type the type of Decoration to construct (must be a key to Decoration.PROPERTY_TABLE)
+     * @param {Vector} pos the position of the bottom left corner of the Decoration (in world space) 
+     */
+    constructor(type, pos) {
+        const properties = Decoration.PROPERTY_TABLE[type];
```

Comment:
May be a good idea to throw an error/exception here, with a clear explanation, when an improper "type" is passed in.

Reply:
I agree. I assume type would be an int and there would be constants like CHAD_HOUSE or such and such for the keys.

Reply:
Throwing an exception for an invalid argument is a good idea, thanks guys. @calebkrauter I'm not so sure that named integer constants are needed in this case. We used those in other classes (e.g. Slime) since they were convenient for delineating which sprites to draw. For this class, I think that strings are sufficient but am open to further discussion.

Reply:
Sounds good! That is my mistake. I am a bit new to this type of mapping and mistook the "type" for an int. Good work!

Reply:
No worries! Using named integers would still be a valid approach.